docs/UserGuide.md

```diff
@@ -100,180 +93,245 @@ Do have a look at our Table of Contents to locate the help you need!
+This box tells you the acceptable fields that you can provide for a particular command:
 
-### Adding a person: `add`
+<div markdown="block" class="alert alert-primary">
+**:mag_right: Format:**
+`add n/NAME p/PHONE_NUMBER [t/TAG]…​ [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​ [fb/FACEBOOK] [ig/INSTAGRAM] [tele/TELEGRAM] [tiktok/TIKTOK] [twitter/TWITTER]`
+* Detail 1
+* Detail 2
+* Detail 3
+</div>
 
-Adds a person to the address book.
+If you still aren't sure of how a particular command works, this box shows you examples that you can try out:
 
-Format: `add n/NAME p/PHONE_NUMBER [t/TAG]…​ [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​ [fb/FACEBOOK] [ig/INSTAGRAM] [tele/TELEGRAM] [tiktok/TIKTOK] [twitter/TWITTER]`
+<div markdown="block" class="alert alert-success">
+**:heavy_check_mark: Example:** `add n/Alex Yeoh p/87438807 t/friends fb/alex.yeoh ig/alex.yeoh tele/alyeoh tiktok/alex.yeoh date/birthday:2000-01-01:yearly`
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags and dates (including 0).
+Adds a new contact called "Alex Yeoh" with phone number "87438807" to SociaLite.
 </div>
 
-Examples:
-* `add n/Alex Yeoh p/87438807 t/friends fb/alex.yeoh ig/alex.yeoh tele/alyeoh tiktok/alex.yeoh date/birthday:2000-01-01:yearly`
-* `add n/Charlotte Oliveiro p/93210284 t/neighbours fb/charlotte.oli tele/olichar twitter/charolive date/wedding anniversary:2020-01-01:yearly`
+--------------------------------------------------------------------------------------------------------------------
 
-<br>
+## Commands <a name="commands"></a>
+
+This section provides you with comprehensive details of all the commands available on SociaLite. 
+Simply click on any command below to learn more about it!
+
+| `COMMAND` | Description |
+| --------- | ----------- |
+| [`add`](#add) | Add a contact |
+| [`list`](#list) | List all contacts |
+| [`CLICK`](#click) | Access a contact's social media page |
+| [`↑`/`↓`](#scroll) | Track history of commands |
+| [`edit`](#edit) | Edit a contact |
+| [`edit INDEX [t/TAG]`](#edit_tag) | Create / Edit tags for existing contacts |
+| [`edit INDEX t/`](#delete_tag) | Delete all tags from a contact |
+| [`edit INDEX [p/PLATFORM]`](#edit_platform) | Modify social media handles for a contact |
+| [`edit INDEX [date/NAME:YYYY-MM-DD[:monthly/:yearly]]…​`](#edit_dates) | Modify dates for a contact |
+| [`delete`](#delete) | Delete a contact |
+| [`clear`](#clear) | Delete all contacts |
+| [`find`](#find) | Find a contact by name |
+| [`find t/TAG`](#find_tag) | Find a contact through associated tags |
+| [`find p/PLATFORM`](#find_platform) | Find a contact through associated social media platforms |
+| [`picture`](#picture) | Modify profile picture of a contact |
+| [`share`](#share) | Copy contact information to system clipboard |
+| [`pin`](#pin) | Pin contact card on the top of the person list |
+| [`unpin`](#unpin) | Unpin contact card from the top of the person list |
```

Comment:
I was thinking maybe these few commands should be `share INDEX`, `pin INDEX`, `remark r/REMARK` and so on! Also to standardise with the earlier sections.

Reply:
Got it! I'll changesthem to `picture INDEX`, `share INDEX`, `pin INDEX`, `unpin INDEX`, `remark r/REMARK`!